common/ocp-config/ds-component-environment/ds-component-oauthproxy.yml

```diff
@@ -128,7 +128,7 @@ objects:
         spec:
           containers:
             - name: oauth-proxy
-              image: openshift/oauth-proxy:latest
+              image: registry.redhat.io/openshift4/ose-oauth-proxy:latest
```

Comment:
as said in the comments, let's wait (or not, fix later) for the change to happen in parallel, where we plan to have that image in the `foo-cd` namespace - which will solve the issue of image promotion imports (ODS releases)

Reply:
Wait how long? ^^.
Alternatively, I can switch back to the imag ein the openshift namespace

Reply:
latest next release, which should be next month, right @braisvq1996 ? so this change should be covered

Reply:
Just need to know which one. I just tested it with the previously used: openshift/oauth-proxy:latest. It also works. 
I just had in the back of my mind, that the redhat one is preferred, but have no preference, myself.

Reply:
@gerardcl, @braisvq1996, what image should it be? It works with both!